04-ruby-mvc-sinatra/active-record-intro/readme.md

```diff
@@ -387,28 +395,28 @@ class Instructor < ActiveRecord::Base
 end
 ```
 
-In my `models/student.rb`:
+Now In my `models/student.rb` we have to reflect the association:
 
 ```ruby
 class Student < ActiveRecord::Base
   belongs_to :instructor
 end
 ```
-> note the plurality of students and singularity of instructor
+> note the plurality of `students` and singularity of `instructor`
 
-we also need to include the `models/instructor.rb` file into our `app.rb` so in `app.rb` we need to add
+We also need to include the `models/instructor.rb` file into our `app.rb` so in `app.rb` we need to add
 
 ```ruby
 require_relative "models/instructor"
 ```
 
 ### Updating Class Defintions - Hospital (You do 5m)
```

Comment:
It might be worth splitting up this lesson into two discrete lessons a la - https://github.com/ga-dc/curriculum/blob/master/04-ruby-mvc-sinatra/object_oriented_programming_in_ruby/readme.md